soccer_ipm/soccer_ipm/msgs/ball.py

```diff
@@ -0,0 +1,53 @@
+from ipm_interfaces.msg import Point2DStamped
+from ipm_library.exceptions import NoIntersectionError
+from ipm_library.ipm import IPM
+from rclpy.impl.rcutils_logger import RcutilsLogger
+from soccer_ipm.utils import create_field_plane
+import soccer_vision_2d_msgs.msg as sv2dm
+import soccer_vision_3d_msgs.msg as sv3dm
+
+
+def map_ball_array(
+        msg: sv2dm.BallArray,
+        ipm: IPM,
+        output_frame: str,
+        logger: RcutilsLogger,
+        ball_diameter: float) -> sv3dm.BallArray:
+    """
+    Map a given array of 2D ball detections onto the field plane.
```

Comment:
To me, ``field plane`` would mean the ground. For the ball, it's z=ball_diameter, so it won't be field plane.